### PR TITLE
TTT: Update Turkish

### PR DESCRIPTION
**Pull request by NovaDiablox**
I can't believe I missed this one.

### CODE REVIEW

File: garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua [hunk 1]
```diff
@@ -213,7 +213,7 @@ L.sb_mia = "Eylem Eksik"
 L.sb_confirmed = "Onaylanmış Ölü"
 
 L.sb_ping = "Gecikme"
-L.sb_deaths = "Ölümler"
+L.sb_deaths = "Ölüm"
```
Comment (robotboy655) [inline garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua:216]: I don't think this change is correct? The English string for this is plural, not singular.

https://github.com/Facepunch/garrysmod/blob/33740f86eea326dbc7f84104e993f7c769c65329/garrysmod/gamemodes/terrortown/gamemode/lang/english.lua#L216

Comment (NovaDiablox) [reply garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua:216]: Its better this way. The change mimics the CS2 scoreboard and its generally used singular if its in a scoreboard.

Comment (NovaDiablox) [reply garrysmod/gamemodes/terrortown/gamemode/lang/turkish.lua:216]: image
Here is an example.

**Comment by NovaDiablox** [comment]
indir
Another example.

**Comment by robotboy655** [comment]
I am not sure sample size of 1 other game is a good argument, but I guess I don't speak Turkish, so you probably know better.

**Comment by NovaDiablox** [comment]
> I am not sure sample size of 1 other game is a good argument, but I guess I don't speak Turkish, so you probably know better.

image
Here is an another example from CS:S. Thanks for merging.

**Pull request merged**